src/engine/renderer/tr_image.cpp

```diff
@@ -170,286 +580,270 @@ void R_ImageList_f()
 	{
 		image = (image_t*) Com_GrowListElement( &tr.images, i );
 		char buffer[ MAX_TOKEN_CHARS ];
+		char imageType[ MAX_TOKEN_CHARS ];
```

Comment:
Just use `std::string`

Reply:
Still `TOTO`.

Reply:
Hmm, there is still usage of `Com_sprintf`, not sure how to handle that with `std::string`.

Reply:
`.c_str()` to get a char*

Reply:
I still don't know how to do this with `std::string`:

```c
Com_sprintf( imageType, sizeof( imageType ),  "%5i ", image->type );
```

Reply:
Com_sprintf -> Str::Format